.github/workflows/ci.yml

```diff
@@ -227,333 +201,201 @@ jobs:
-        uses: actions/checkout@v3
-        with:
-          fetch-depth: 0
-      - name: before step
-        if: ${{ inputs.gitleaks_before_step_command != '' }}
-        run: |
-          ${{ inputs.gitleaks_before_step_command}}
-      - name: Gitleaks scan
-        uses: gitleaks/gitleaks-action@v1.6.0
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      - name: after step
-        if: ${{ inputs.gitleaks_after_step_command != '' }}
+          ACCESS_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          PR_NUMBER: ${{ github.event.number }}
+          PROJECT_NAME: ${{ github.event.repository.name }}
+          PROJECT_OWNER: ${{ github.repository_owner }}
+        if: ${{ always() && !contains( inputs.excluded_jobs, 'licenseScan' ) }}
         run: |
-          ${{ inputs.gitleaks_after_step_command }}
+          /root/executables/ci-utility check --sbom bom.xml --allowedLicences ${{ inputs.allowedLicenses }}          
 
-  lint:
-    name: eslint scan to find linting issues
-    if: ${{ !contains( inputs.excluded_jobs, 'lint' ) && (github.actor != 'dependabot[bot]') }}
-    runs-on: ubuntu-latest
-    permissions: write-all
-    needs: caching
-    container:
-      image: public.ecr.aws/studiographene/ci:node-20-alpine
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v3
-        with:
-          ref: ${{ github.REF }}
-      - name: before step
-        if: ${{ inputs.lint_before_step_command != '' }}
-        run: |
-          ${{ inputs.lint_before_step_command}}
-      - name: Cache npm modules
-        uses: actions/cache@v3
-        if: inputs.package_manager == 'npm'
-        with:
-          path: ~/.npm
-          key: ${{ runner.os }}-npm-${{ hashFiles('**/package-lock.lock') }}
-          restore-keys: |
-            ${{ runner.os }}-npm-
-      - name: Cache yarn modules
-        uses: actions/cache@v3
-        if: inputs.package_manager == 'yarn'
-        with:
-          path: /usr/local/share/.cache/yarn
-          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
-          restore-keys: |
-            ${{ runner.os }}-yarn-
-      - name: Cache pnpm modules
-        uses: actions/cache@v3
-        if: inputs.package_manager == 'pnpm'
-        with:
-          path: /__w/.pnpm-store
-          key: ${{ runner.os }}-pnpm-${{ hashFiles('**/pnpm-lock.yaml') }}
-          restore-keys: |
-            ${{ runner.os }}-pnpm-
-      - name: Install dependencies
-        run: |
-          ${{ inputs.package_manager }} install
-        env:
-          NPM_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      - name: Eslint scan
-        run: |
-          ${{ inputs.lint_command }}
-      - name: after step
-        if: ${{ inputs.lint_after_step_command != '' }}
-        run: |
-          ${{ inputs.lint_after_step_command }}
+      - name: Dependency Scan using Google OSV
+        id: dependency_scan
+        if: ${{ always() && !contains( inputs.excluded_jobs, 'dependency_scan' ) }}
```

Comment:
aren't dependabot[bot], action bot, issue comment checks needed here?

Reply:
Actually that is the step, the main job is configured with the check for the bots